### DISCUSSION TITLE
TheHive Authentication issues

### DISCUSSION THREAD

**Discussion by BustedSec** [Security-Onion-Solutions/securityonion#2180] [Unsupported Versions] [answered]
Running SO 2.3.10 on ubuntu - Standalone install - ESXi Host - 1TB HDD, 16 cores, 12GB RAM

Going to TheHive for the first time results in a page saying that it must be updated, I click the button, it updates, asks me to create a user account. My email is pre-populated in the Name field. I enter a user name and password. I attempt to then use that same user/pass to log in. Authentication Failed, access denied. 

tried to create new account via CLI with so-user-add email@domain.com
Unable to add user to TheHive; user might already exist
{"type":"AuthenticationError","message":"Authentication failure"}

Tried deleting thehive index with curl,  reboot, go back to TheHive page - same "need to update" page shows - enter user name and password again, new ones this time, same thing - they don't work.

**Comment by weslambert** [comment, marked as answer]
It sounds like TheHive didn't get correctly initialized during setup.  You shouldn't need to update TheHive ES instance manually.  We are working on a script to fix this intermittent issue.  I will have to perform some testing, which make take some time, otherwise, you could try installation a fresh install to see if you get the same result.
